pkg/certs/cert-inspection/certgraphanalysis/collector.go

```diff
@@ -257,3 +257,45 @@ func MergePKILists(ctx context.Context, first, second *certgraphapi.PKIList) *ce
 		OnDiskResourceData:          onDiskResourceData,
 	}
 }
+
+func GetBootstrapHostname(ctx context.Context, kubeClient kubernetes.Interface) (string, error) {
+	bootstrapIP := ""
+	etcdConfigMaps, err := kubeClient.CoreV1().ConfigMaps("openshift-etcd").List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return "", err
+	}
+	for _, cm := range etcdConfigMaps.Items {
+		annotation, ok := cm.Annotations["alpha.installer.openshift.io/etcd-bootstrap"]
+		if ok {
+			bootstrapIP = annotation
+			break
+		}
+	}
+
```

Comment:
return here if the bootstrapIP is empty?

Reply:
We don't need the bootstrap IP, secret names have k8s name set there ("Peer cert for node ip-10-11-12-13", not "Peer cert for node 10.11.12.13")

Reply:
Done